src/main/java/com/matt/forgehax/asm/utils/remapping/MCPMappingLoader.java

```diff
@@ -22,7 +23,7 @@
   private final String baseSrgDir = "{mc_ver}";
   private final String baseMappingDir = "{mc_ver}/{channel}_{map_ver}";
   private final String baseMappingUrl =
-    "http://export.mcpbot.bspk.rs/mcp_{channel}/{map_ver}-{mc_ver}/mcp_{channel}-{map_ver}-{mc_ver}.zip";
+    "https://github.com/ModCoderPack/MCPMappingsArchive/raw/master/mcp_snapshot/20171003-1.12/mcp_snapshot-20171003-1.12.zip";
   private final String baseSrgUrl =
     "http://export.mcpbot.bspk.rs/mcp/{mc_ver}/mcp-{mc_ver}-srg.zip";
```

Comment:
Hope this isn't used

Reply:
is not. (as far as i can see.) (further testing is required.)